### ISSUE TITLE
GetAnotherServerAuthCode always requests email permission

### ISSUE THREAD

**Issue by boerjesieling** [playgameservices/play-games-plugin-for-unity#2409]
Hi,

I am autheticating a user using:

```
        PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
            .RequestServerAuthCode(false)
            .Build();
        ...
        Social.Active.Authenticate(...);

```
this config does NOT requests the email.

When calling 
PlayGamesPlatform.Instance.GetAnotherServerAuthCode(..);
afterwards, it seems that the email is always requested. This results in a popup that requires the user to agree accessing the email (which I dont need/want). 

I traced it down to:

source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs:341
GetAnotherServerAuthCode ->

source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidTokenClient.cs:221
GetAnotherServerAuthCode ->

source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/TokenFragment.java:155
fetchToken is called with requestEmail and requestIdToken hardcoded to "true"

I would expect that the GetAnotherServerAuthCode uses the same config as for the Authentication.

best,
Boerje

**Comment by Noah1911** [comment]
@LadTy as you guessed, the java part must be rebuilt, looks like simple parameter addition which I can pull request to this repository today/sometime this week if developer doesn't make it before me.

**Comment by Noah1911** [comment]
@LadTy @boerjesieling Feel free to test the pull request.

Don't forget to rebuild java lib or download from here: https://uploadfiles.io/9nmke
PS: This file is not officially approved by maintainer/repo owner and I am not responsible for any damage caused by this file...

**Comment by LadTy** [comment]
You're great, thank you. I for now decided to go with an older version anyway (reverted a few other plugins/unity updates as well) just to be on the safe side because I had to put out an update of our game due to other reasons. I sure do appreciate your help though and will be watching for this to be put in the official releases later on.

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.9.58.
